src/cognito/post-confimation.js

```diff
@@ -1,33 +1,84 @@
 'use strict';
-import {DynamoDB} from 'aws-sdk';
 import {CognitoIdentityServiceProvider} from 'aws-sdk';
+import {Organization} from './../db/models/organization';
+import {User} from './../db/models/user';
 
-import * as crypto from 'crypto';
-
-const cognitoidentityserviceprovider = new CognitoIdentityServiceProvider();
-const dynamoDB = new DynamoDB.DocumentClient();
+const cognito = new CognitoIdentityServiceProvider();
+const getAttributeValue = (collection, field) => {
+    const result = collection.UserAttributes.find(
+        (element) => element.Name == field
+    );
+    if (result) return result.Value;
+    else return null;
+};
 
 /**
  * Handles the post confirmation actions
  * @param {*} event
  * @param {*} context
  */
 export const handler = (event, context) => {
-    const params = {
+    let params = {
         UserPoolId: process.env.USER_POOL_ID,
         Username: event.request.userAttributes.email,
     };
 
-    event.request.userAttributes.uuid = crypto.randomBytes(16).toString('hex');
-
-    dynamoDB.put({
-        TableName: process.env.ORGANIZATIONS_TABLE,
-        Item: event.request.userAttributes,
-      }, (err) => {
-        if (err) console.log(err);
-        cognitoidentityserviceprovider.adminDisableUser(params, (err, data) => {
-            if (err) context.done(null, event);
-            else context.done(null, event);
-        });
-      });
+    cognito.adminGetUser(params, (err, user) => {
+        if (err) return context.done(err);
+        if (getAttributeValue(user, 'custom:manualConfirmation') == null) {
```

Comment:
What if we don't have custom:manualConfirmation in the user attributes, we still get null ryt? I don't think this will be an issue as far as we define it manually. Just mentioned.